pandas/tests/frame/indexing/test_getitem.py

```diff
@@ -67,6 +67,13 @@ def test_getitem_sparse_column_return_type_and_dtype(self):
         result = df.loc[:, "A"]
         tm.assert_series_equal(result, expected)
 
+    @pytest.mark.parametrize("dtype", ["bool", "object"])
+    def test_getitem_indexer_empty_series(self, dtype):
+        # GH: 19290
+        df = DataFrame(columns=["a", "b", "c"])
+        result = df[Series([], name="a", dtype=dtype)]
```

Comment:
put the indexer (the empty seris) as a separate statement.

can you parameterize the constructor on np.ndarray as well (we likey have this test already; ideally you can consolidate it with this one).

Reply:
Did the parametrization, test did not exist previously, because array from dtype=object did not work previously